crates/api_models/src/connector_enums.rs

```diff
@@ -0,0 +1,298 @@
+    Payme,
+    Payone,
+    Paypal,
+    Payu,
+    Placetopay,
+    Powertranz,
+    Prophetpay,
+    Rapyd,
+    Razorpay,
+    Shift4,
+    Square,
+    Stax,
+    Stripe,
+    Taxjar,
+    Threedsecureio,
+    //Thunes,
+    Trustpay,
+    Tsys,
+    Volt,
+    Wellsfargo,
+    // Wellsfargopayout,
+    Wise,
+    Worldline,
+    Worldpay,
+    Signifyd,
+    Plaid,
+    Riskified,
+    Zen,
+    Zsl,
+}
+
+impl Connector {
+    #[cfg(feature = "payouts")]
+    pub fn supports_instant_payout(&self, payout_method: Option<PayoutType>) -> bool {
+        matches!(
+            (self, payout_method),
+            (Self::Paypal, Some(PayoutType::Wallet))
+                | (_, Some(PayoutType::Card))
+                | (Self::Adyenplatform, _)
+        )
+    }
+    #[cfg(feature = "payouts")]
+    pub fn supports_create_recipient(&self, payout_method: Option<PayoutType>) -> bool {
+        matches!((self, payout_method), (_, Some(PayoutType::Bank)))
+    }
+    #[cfg(feature = "payouts")]
+    pub fn supports_payout_eligibility(&self, payout_method: Option<PayoutType>) -> bool {
+        matches!((self, payout_method), (_, Some(PayoutType::Card)))
+    }
+    #[cfg(feature = "payouts")]
+    pub fn is_payout_quote_call_required(&self) -> bool {
+        matches!(self, Self::Wise)
+    }
+    #[cfg(feature = "payouts")]
+    pub fn supports_access_token_for_payout(&self, payout_method: Option<PayoutType>) -> bool {
+        matches!((self, payout_method), (Self::Paypal, _))
+    }
+    #[cfg(feature = "payouts")]
+    pub fn supports_vendor_disburse_account_create_for_payout(&self) -> bool {
+        matches!(self, Self::Stripe)
+    }
+    pub fn supports_access_token(&self, payment_method: PaymentMethod) -> bool {
+        matches!(
+            (self, payment_method),
+            (Self::Airwallex, _)
+                | (Self::Deutschebank, _)
+                | (Self::Globalpay, _)
+                | (Self::Paypal, _)
+                | (Self::Payu, _)
+                | (Self::Trustpay, PaymentMethod::BankRedirect)
+                | (Self::Iatapay, _)
+                | (Self::Volt, _)
+                | (Self::Itaubank, _)
+        )
+    }
+    pub fn supports_file_storage_module(&self) -> bool {
+        matches!(self, Self::Stripe | Self::Checkout)
+    }
+    pub fn requires_defend_dispute(&self) -> bool {
+        matches!(self, Self::Checkout)
+    }
+    pub fn is_separate_authentication_supported(&self) -> bool {
+        match self {
+            #[cfg(feature = "dummy_connector")]
+            Self::DummyConnector1
+            | Self::DummyConnector2
+            | Self::DummyConnector3
+            | Self::DummyConnector4
+            | Self::DummyConnector5
+            | Self::DummyConnector6
+            | Self::DummyConnector7 => false,
+            Self::Aci
+            // Add Separate authentication support for connectors
+			// | Self::Fiuu
```

Comment:
Nit: Looks like tabs are being used instead of spaces in some of these lines, in `is_separate_authentication_supported()`, can you address that?